docs/GATK_GERMLINE_README.md

```diff
@@ -1,9 +1,48 @@
 # Kids First DRC Single Sample Genotyping Workflow
 Kids First Data Resource Center Single Sample Genotyping Workflow. This workflow closely mirrors the [Kids First DRC Joint Genotyping Workflow](https://github.com/kids-first/kf-jointgenotyping-workflow/blob/master/workflow/kfdrc-jointgenotyping-refinement-workflow.cwl).
-While the Joint Genotyping Workflow is meant to be used with trios, this workflow is meant for processing single samples.
-The key difference in this pipeline is a change in filtering between when the final VCF is gathered by GATK GatherVcfCloud and when it is annotated by VEP bcftools (see [Kids First DRC Germline SNV Annotation Workflow docs](https://github.com/kids-first/kf-annotation-tools/blob/v1.1.0/docs/GERMLINE_SNV_ANNOT_README.md) ).
-Unlike the Joint Genotyping Workflow, a germline-oriented [GATK hard filtering process](https://gatk.broadinstitute.org/hc/en-us/articles/360035890471-Hard-filtering-germline-short-variants) is performed and CalculateGenotypePosteriors has been removed.
-While somatic samples can be run through this workflow, be wary that the filtering process is specifically tuned for germline data.
+
+While the Joint Genotyping Workflow is meant to be used with whole genome
+sequenced trios, this workflow is meant for processing single samples from any
+sequencing experiment. The key difference between the different approaches is
+the filtering process.
+
+While somatic samples can be run through this workflow, be wary that the
+filtering process is specifically tuned for germline data.
+
+## GATK Genotype Site-Level Filtering
+
+Coming out of the GATK Genotyping process, site-level filtering must be done to
+remove variants that might adversely affect downstream analysis.
+
+GATK provides many different approaches to filtering:
+- Variant Quality Score Recalibration (VQSR)
+- CNNScoreVariants/NVScoreVariants
+- Variant Extract-Train-Score (VETS)
+- Hard Filtering
+
+The first three are all complex model-based approaches that attempt to infer
+cutoff points based on the data provided. Hard Filtering is manually setting
+thresholds and removing variants that fail to meet those thresholds. For this
+workflow, we only make use of VQSR and Hard Filtering at this time.
+
+VQSR, being a model based approach, needs sufficient data to construct that
+model. Normally in the joint filtering context, this means having hundreds of
+samples. According to the documentation: "it is not suitable for some
+small-scale experiments, such as targeted gene panels or exome studies with
+fewer than 30 exomes." Therefore, VQSR is only activated in this workflow when
+the input gVCFs for this workflow come from whole genome sequencing experiments
+or when the user provides 30 or more exome gVCFs.
```

Comment:
If you provide 30+ exomes, target, do you get a joint called VCF, or individual calls with VQSR?

Reply:
They all get jointly processed. 30+ exomes will go to VQSR, 30+ targeted will still be hard filtered.

Targeted seq just is too high depth and too low data to work properly in VQSR.